types/puppeteer/index.d.ts

```diff
@@ -473,6 +474,13 @@ export interface NavigationOptions {
    * @default load Navigation is consider when the `load` event is fired.
    */
   waitUntil?: LoadEvent | LoadEvent[];
+
+  /**
+   * Referer header value.
+   * If provided it will take preference over the referer header value set by
+   * [page.setExtraHTTPHeaders()](#pagesetextrahttpheadersheaders).
+   */
+  referer?: string;
```

Comment:
This interface is used for `goBack(options)` which doesn't have this property according to the  api docs: https://github.com/GoogleChrome/puppeteer/blob/v1.8.0/docs/api.md#pagegobackoptions

Reply:
That's annoying, I will create a new interface then.